src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 20;
+    private static final long VALID_ID = 123883022L;
+    private static final String VALID_LOGIN = "UserName";
+    private static final String VALID_PASSWORD = "BestPasswordEver123";
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(VALID_AGE);
+        user.setId(VALID_ID);
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+    }
+
+    @Test
+    void register_lowerThanExpectedAge_NotOk() {
+        int age = 17;
+        user.setAge(age);
+        assertTrue(user.getAge() < 18);
+    }
+
+    @Test
+    void register_equalsToExpectedAge_Ok() {
+        int age = 18;
+        user.setAge(age);
+        assertEquals(user.getAge(), 18);
+    }
+
+    @Test
+    void register_graterThanExpectedAge_Ok() {
+        int age = 19;
+        user.setAge(age);
+        assertFalse(user.getAge() < 18);
+    }
```

Comment:
The test `register_graterThanExpectedAge_Ok` should include a call to `registrationService.register(user)` to ensure that no exception is thrown when the age is greater than 18.